src/BlockEntities/BannerEntity.h

```diff
@@ -25,16 +25,20 @@ class cBannerEntity :
 
 public:
 
-	cBannerEntity(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, Vector3i a_Pos, cWorld * a_World);
-	cBannerEntity(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, Vector3i a_Pos, cWorld * a_World, unsigned char a_BaseColor);
+	cBannerEntity(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, Vector3i a_Pos, cWorld * a_World, unsigned char a_BaseColor = 15, AString a_CustomName = "");
```

Comment:
Previously the default BaseColor was 1, now it's 15?

Reply:
It's a bit pedantic, I guess :-). Banner colours are inverted 0-15 so 15 is white (0 is black and 1 is just somewhat random). That said crafting a banner always sets damage (i.e. colour) on the crafted item, loading by WSSAnvil always sets the colour to whatever was saved and placing a banner always does a SetBaseColor using the source item's damage. Unless there's a way to magic a banner into existence from Lua (I haven't checked) those two defaulted args have no real purpose and could be vacuumed up?